capsules/extra/src/tickv.rs

```diff
@@ -472,6 +472,9 @@ impl<'a, F: Flash, H: Hasher<'a, 8>, const PAGE_SIZE: usize> flash::Client<F>
                             );
                         });
                     }
+                    Err(tickv::error_codes::ErrorCode::ReadNotReady(_)) => {
+                        // Need to do another flash read.
```

Comment:
```suggestion
                        // Need to do another flash read.
                        //
                        // `self.operation` will still be `GetKey`, so this will automatically
                        // be retried by the primary state machine.
```
I think this is why this works? I thought this was a "todo: need to do another read" looking at this at first — the implicit retry isn't terribly obvious.